alf/environments/utils.py

```diff
@@ -173,6 +174,10 @@ def create_environment(env_name='CartPole-v0',
 
     """
 
+    # Use GPU before loading environments so cluster_train idle will not show
+    # the GPU
+    tmp = torch.zeros((32, ))
```

Comment:
I understand the motivation of this. But the comment seems unrelated to ALF?

Reply:
Updated the comment